tests/i18n/test_extraction.py

```diff
@@ -221,6 +236,7 @@ def test_template_message_context_extractor(self):
         Ensure that message contexts are correctly extracted for the
         {% trans %} and {% blocktrans %} template tags.
         Refs #14806.
+        Refs #25297.
```

Comment:
The ticket reference isn't really useful I think (in general we only include if there's some information in the ticket the can't be easily expressed in comments).